src/main/java/com/rakovets/course/java/core/practice/exception_handling/Task3.java

```diff
@@ -0,0 +1,19 @@
+package com.rakovets.course.java.core.practice.exception_handling;
+
+import com.rakovets.course.java.core.practice.exception_handling.exception.FactorialException;
+
+public class Task3 {
+    int result = 1;
+
+    void calculate(int number) throws FactorialException{
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"